face_api_dataset/dataset.py

```diff
@@ -58,21 +62,53 @@ class Modality(Enum):
     
     **Type**: `ndarray[uint16]`. **Channels**: 1.
     """
-    LANDMARKS = auto()
+    LANDMARKS_IBUG68 = auto()
+    """
+    iBUG-68 landmarks. Each landmark is given by name and two coordinates (name, x,y) in pixels.
+    Each keypoint is a 2D projection of a 3D landmark. 
+    
+    **Type**: `List[Type[str, int, int]`.
+    """
+    LANDMARKS_CONTOUR_IBUG68 = auto()
     """
-    iBUG-68 landmarks. Each landmark is given by two coordinates (x,y) in pixels.
+    iBUG-68 contour landmarks. Each landmark is given by two coordinates (name, x,y) in pixels.
+    Each keypoint is defined in a similar manner to human labelers marking 2D face kepoints.
     
-    **Type**: `ndarray[float64]`. **Dimensions**: (68, 2).
+    **Type**: `List[Type[str, int, int]`. Should have 68 points.
```

Comment:
Should have no more than 68 points (only visible points are included).

Reply:
fixed